generators/chipyard/src/main/scala/HarnessBinders.scala

```diff
@@ -324,6 +325,21 @@ class WithSimDromajoBridge extends ComposeHarnessBinder({
 
 class WithTieOffCustomBootPin extends OverrideHarnessBinder({
   (system: CanHavePeripheryCustomBootPin, th: HasHarnessSignalReferences, ports: Seq[Bool]) => {
-    ports.foreach(_ := false.B)
+    val pin = PlusArg("custom_boot_pin", width=1)
```

Comment:
Nice

Reply:
We should figure out how to get PlusArg to work with FireSim. Seems like a chill ugrad project.

Reply:
I'll add it. Additionally, I think this harness binder name should probably change since it isn't strictly a tie-off anymore. Maybe something like `WithCustomBootPinPlusArg`?